### PR TITLE
dullahan/smolraptor bug fixes and syndicate + clownborg additions!

### PR DESCRIPTION
**Pull request by Asher-42**
adds in:
clown borg

dully sprites to syndie and their sub modules

fixes: panel opening issues and some recoding
## Changelog
:cl:
add: Dullahan Clownborg
add: panel covers to dullahan and smol raptors
add: dullahan syndicate sprite to only medic
/:cl:
image
dreammaker_V3GZCn0rfI

### CODE REVIEW

**Comment by Kashargul** [comment]
We have the syndicate V2 sprite, it's in use for the combat medic.

File: modular_chomp/code/modules/mob/living/silicon/robot/sprites/civilian.dm [hunk 1]
```diff
@@ -23,22 +23,22 @@
 /datum/robot_sprite/dogborg/tall/service/dullahanservice
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/living/silicon/robot/sprites/civilian.dm:23]: We have multiple of the same type, move the DMI and offset to a parent path.

File: modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm [hunk 2]
```diff
@@ -0,0 +1,72 @@
+/datum/robot_sprite/dogborg/syndie/v2dogborgsyndie
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm:1]: This V2 sprite was exclusively meant and is in use for the combat medic.

File: modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	name = "Dogborg V2"
+	sprite_icon = 'modular_chomp/icons/mob/v2dogborgs_ch/v2dogborg_sprite.dmi'
+	sprite_icon_state = "vale"
+	has_eye_light_sprites = TRUE
+	has_custom_open_sprites = TRUE
+	has_vore_belly_sprites = TRUE
+	rest_sprite_options = list("Default", "Sit")
+
+
+/datum/robot_sprite/dogborg/tall/combat_medic/dullahancombatmedic
```
Comment (Kashargul) [inline modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm:35]: Why are they all named Dullahan Combat Medic? Dullahan is enough.

**Review by Kashargul — CHANGES_REQUESTED** [review]

File: modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm (hunk 3 above)
Comment (Asher-42) [reply modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm:35]: 2 am coding, il edit / delete

File: modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm (hunk 2 above)
Comment (Asher-42) [reply modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm:1]: deleting

File: modular_chomp/code/modules/mob/living/silicon/robot/sprites/civilian.dm (hunk 1 above)
Comment (Asher-42) [reply modular_chomp/code/modules/mob/living/silicon/robot/sprites/civilian.dm:23]: where should i move the dmi? i dont understand this part

**Review by Kashargul — CHANGES_REQUESTED** [review]
Still a lot of duplicated states in the DMI files which shouldn't exist.

**Review by Kashargul — APPROVED** [review]

**Pull request merged**